packages/admin/src/common/upload.ts

```diff
@@ -0,0 +1,17 @@
+import { createConfig } from "../config";
+import { GQLEmailCampaignContentScopeInput } from "../graphql.generated";
+
+const config = createConfig();
```

Comment:
There shouldn't be a createConfig function in the library. This should come via injection (= a React provider) from the application.